src/containers/pages/IRS/JurisdictionsReport/index.tsx

```diff
@@ -92,9 +92,11 @@ const JurisdictionReport = (props: GenericJurisdictionProps & RouteComponentProp
             { jurisdiction_depth: true }
           );
         }
-        await service(slice, fetchJurisdictionsParams).then((result: GenericJurisdiction[]) =>
-          fetchJurisdictions(slice, result)
-        );
+        await service(slice, fetchJurisdictionsParams).then((result: GenericJurisdiction[]) => {
+          if (result) {
+            return fetchJurisdictions(slice, result);
```

Comment:
what about making this check the responsibility of `fetchJurisdictions`?

Reply:
Oh that's a good point, then it would protect it everywhere it's being called 👍

Reply:
cool, and future extension could be having it return an object on error than caller knows how to render for user... but not now :)

Reply:
Actually, if the results are being passed in as `undefined`, then [this line](https://github.com/onaio/reveal-frontend/blob/master/src/store/ducks/generic/jurisdictions.ts#L113) should actually be preventing `undefined` [from being iterated upon](https://github.com/onaio/reveal-frontend/blob/master/src/store/ducks/generic/jurisdictions.ts#L116).